yarn.lock

```diff
@@ -1,18 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
We'll be rebasing often, and I've found it a pain to rebase `.lock` files.

Actually, IMHO it isn't worth the headache of committing a `.lock` file for websites. In the case of etcd.io we only have a small set of dev dependencies. My suggestion would be to drop the `.lock` file from `master`, and propagate that change to here.

Reply:
I've found the same -- I'll remove the lock files before merging.